### ISSUE TITLE
idea: add rss feeds to our slacks

### ISSUE THREAD

**Issue by drmowinckels** [rladies/awesome-rladies-blogs#68]
We are getting so many awesome feeds through this repo, and in both the organiser slack and the community slack there are #rladiesblogs (variations on that spelling) where feeds from community member's blog come. 

We should add the feeds from here to those slacks! We can either do that manually through the `/feed` command, or find some other nifty way of doing it.

**Comment by cosimameyer** [comment]
I love the idea! 💜 I remember that I also tried to add my blog with `/feed` but failed 😄 I think if we either add the existing blogs manually or find an automatic pipeline (even better), that would be fantastic!

**Comment by drmowinckels** [comment]
agree, something automatic would be great. Let explore that when we have the time. Likely the best option is to make some slack bot, but uncertain. I have colleagues who have made slack bots claiming it isnt too hard :P

**Comment by drmowinckels** [comment]
ok, so I made an action that is getting somewhere, but not finishing the job correctly.

https://github.com/rladies/awesome-rladies-blogs/blob/main/.github/workflows/slack-feed.yaml

It takes an url as an input (so we can trigger it for instance after merge to main), and it posts a message to the slack channel `/feed url` . Problem is, Slack is not recognising it as a command, just as a message, and so the feed is not added.

I've been looking for a "command" webapi endpoint from slack, but cant seem to find it. But we have something to base ourselves off of.

**Comment by cosimameyer** [comment]
That looks cool, thanks so much for the headstart! I also looked into it yesterday and found [this](https://stackoverflow.com/questions/39829741/execute-slash-command-as-slack-bot) - I’m not sure whether sending slash commands as a bot still works (SO says "no" but I’m hoping for a customized workaround 😄)

**Comment by drmowinckels** [comment]
Yeah, I have been poking around for answers, and there is an undocumented webapi command `chat.command` that _used_ to work. It technically still works, but you'll need a legacy token, which we cannot get a hold of :/

**Comment by cosimameyer** [comment]
I know :( I got a reply and sent it to you on Slack - it's not really helpful but probably you can access one of the links? (most likely the second one)

**Comment by cosimameyer** [comment]
I didn't fully evaluate them but I kept looking for a solution and wanted to leave my "finds" here to not forget about them:

- [imitate the behavior of the RSS app (not sure how much work this eventually will be)](https://stackoverflow.com/a/76355359)
- [use a specific user’s token (not directly what they say but what I thought of; I think it doesn't work because it still uses the endpoints programmatically but I'm wondering if there are permission differences when it comes to users vs bots)](https://stackoverflow.com/questions/73071161/is-there-a-way-to-send-message-as-a-user-not-as-a-bot-when-using-slash-command)
- [attempt of serializing and deserializing input to define the type of output](https://stackoverflow.com/questions/75114241/slack-api-slash-command-sent-to-asp-net-core-mvc-web-app-issue)

I also checked these repositories:
- [undocumented commands in Slack API](https://github.com/ErikKalkoken/slackApiDoc/issues)
- [Python SDK; but there are of course also others :)](https://github.com/slackapi/python-slack-sdk)
